var/spack/repos/builtin/packages/hadoop/package.py

```diff
@@ -3,7 +3,7 @@
 # Produced at the Lawrence Livermore National Laboratory.
 #
 # This file is part of Spack.
-# Created by Alfredo Gimenez, gimenez1@llnl.gov, All rights reserved.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
```

Comment:
Yeah it is, though I do not like it.  This is the license template the lab wants.  I am trying to get rid of the requirement for this but the wheels of licensing policy turn slowly.  I'd rather just have a single LGPL notice at the top and not care about copyright (as we will probably just keep Spack LGPL and don't need to relicense).  I am thankful that we have `git blame` and good public github stats to give people credit.  The license is just an artifact of the original release from LLNL.